Eask

```diff
@@ -9,7 +9,7 @@
 
 (package-file "flymake-perlcritic.el")
 
-(script "test" "echo \"Error: no test specified\" && exit 1")
+(script "test" "emacs -batch -l ert -l flymake-perlcritic.el -l test/flymake-perlcritic-test.el -f ert-run-tests-batch-and-exit")
```

Comment:
I believe `eask test ./test/.*.el` does the same thing. 🤔

Reply:
I’ve updated the setup to use `eask test ert ....`.
(N.B. all required files must be explicitly listed, as they are loaded [using `load-file`](https://github.com/emacs-eask/cli/blob/8324d7a458b46593d7c649cb32a537ba87352c3a/lisp/test/ert.el#L56).)